fe-public/package.json

```diff
@@ -10,15 +10,15 @@
     "astro": "astro"
   },
   "dependencies": {
-    "@astrojs/tailwind": "^5.0.0",
     "astro": "^3.0.10",
     "astro-seo": "^0.8.0",
     "gray-matter": "^4.0.3",
-    "tailwindcss": "^3.3.3",
     "node-ical": "^0.16.1",
+    "tailwindcss": "^3.3.3",
     "uuid": "^9.0.1 "
   },
   "devDependencies": {
+    "@astrojs/tailwind": "^5.0.2",
     "@types/uuid": "^9.0.4",
     "node-ical": "^0.16.1",
     "tailwindcss": "^3.3.3"
```

Comment:
Im not sure how this would go, maybe try to run `npm run build` and then `cd dist` and then run `npx serve .` To see if it caused any issues.

Reply:
yeah i just restarted my thing and it worked, ima remove the extra  ie connects off the burger